templates/default/zip-background.sh.erb

```diff
@@ -21,6 +21,7 @@ cd "$1"
 freespace=$(df "$PWD" | awk '/[0-9]%/{print $(NF-2)}')
 freespace=$((freespace * 1000))
 
+hdfsFile=$2
```

Comment:
Can you put this between quotes - `"$2"`

Reply:
Fixed